### ISSUE TITLE
Passive voice is sometimes just sonkeigo.

### ISSUE THREAD

**Issue by nicolasmaia** [birchill/10ten-ja-reader#187]
When parsing the following sentence,

私が尊敬してやまない木村先生が賞を受けられた

Rikai says 受けられた is possibly in the passive voice, but in fact it's sonkeigo. This also happens to common expressions when talking to superiors, like 思われる. Since there's no reliable way to distinguish between the two without context, Rikai could at least indicate it might also be respectful language.

**Comment by birtles** [comment]
You're right. This would be good to point out. It does get a bit wordy though. Currently for 受けられた it says "< potential or passive < past".

I guess this would become "< potential or passive or respectful < past"?

**Comment by nicolasmaia** [comment]
I think that works!
